### DISCUSSION TITLE
Where is the current code point table?

### DISCUSSION THREAD

**Discussion by Gitoffthelawn** [ryanoasis/nerd-fonts#1737] [Q&A]
I've been reading all the wiki, FAQ, and other pages regarding the Nerd Font project. Well done!

I thought I came across a comprehensive up-to-date code point table during my readings.  But I can't find it now.  All I can find is this proposed one from an issue report: https://github.com/ryanoasis/nerd-fonts/issues/365

Is there a current code point table published somewhere for Nerd Fonts?

**Comment by Finii** [comment]
https://github.com/ryanoasis/nerd-fonts/blob/master/glyphnames.json

**Reply by Finii** [reply]
Well, that is not a table but a database ;-)

Kind of a hand maintained abstract table is https://github.com/ryanoasis/nerd-fonts/wiki/Glyph-Sets-and-Code-Points

**Reply by Finii** [reply]
And there are the (hopefully be dropped sometime, intermediate files `bin/script/lib/i_*.sh` with some info in the head.

And then there are icons that are mentioned nowhere, because they do not belong to any set, like the box drawing glyphs on codepoints `25xx` etc.

**Reply by Gitoffthelawn** [reply]
Thanks. It was definitely a table, so it must have been [this one](https://github.com/ryanoasis/nerd-fonts/wiki/Glyph-Sets-and-Code-Points) that you graciously provided, or the one from #365... unless there is another table lingering around somewhere!

I find such tables to be very [*convientient*](https://github.com/ryanoasis/nerd-fonts/issues/1736#issuecomment-2481408706). 😄
